staging/vhost-device-sound/src/device.rs

```diff
@@ -128,10 +128,13 @@ impl VhostUserSoundThread {
         vring: &VringRwLock,
         audio_backend: &RwLock<Box<dyn AudioBackend + Send + Sync>>,
     ) -> IoResult<bool> {
+        let Some(ref atomic_mem) = self.mem else {
```

Comment:
I'd move the changes to the code related to the new ` NoMemoryConfigured` in a separate commit.

Reply:
done